src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -107,7 +107,7 @@ function getMediaSourceHtml(user, item, version) {
         if (stream.CodecTag) {
             attributes.push(createAttribute(globalize.translate('MediaInfoCodecTag'), stream.CodecTag));
         }
-        if (stream.IsAVC != null) {
+        if (stream.Type === 'Video' && stream.IsAVC != null) {
```

Comment:
Although this change looks fine, I see that `IsAVC` is supposed to be nullable. Did we accidentally set it for non-video streams?

Reply:
At least on my install, yes

Reply:
It probably becomes `false` here https://github.com/jellyfin/jellyfin/blob/72077490447d3ff588563cc7f8aa705e1a7e9ecc/Emby.Server.Implementations/Data/SqliteItemRepository.cs#L5627
`TryGetBoolean` doesn't seem to return `null` if the value in the DB is `NULL`.
Wouldn't `result` be assigned the value `false` instead of `null`?
https://github.com/jellyfin/jellyfin/blob/72077490447d3ff588563cc7f8aa705e1a7e9ecc/Emby.Server.Implementations/Data/SqliteExtensions.cs#L151

Reply:
> TryGetBoolean doesn't seem to return null if the value in the DB is NULL.
Wouldn't result be assigned the value false instead of null?

No, it doesn't assign any value as the Try method returns false bc it's null inside of the DB.
We don't have to assign anything as the default value is `null`

Reply:
This means that we saved `false` in the database. I checked my database - not a single `NULL`.